test/components/app.scss

```diff
@@ -2,4 +2,4 @@
  * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
  */
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
-@fa-font-path: "@{bedrock-module-path}/@fortawesome/fontawesome-free/webfonts";
+$fa-font-path: "@{bedrock-module-path}/@fortawesome/fontawesome-free/webfonts";
```

Comment:
`bedrock-module-path` is a var here that needs to be updated to use the scss syntax.

We also need support for the var in scss via `bedrock-views` or some other method (like always setting it in top-level applications). See: https://github.com/digitalbazaar/bedrock-views/issues/43

@davidlehn, @gannan08, @mattcollier -- do any of you guys have thoughts on what we want to do moving forward here? What we may do to remove the need to specify frontend dependencies could also be impacted (or help solve this issue).

Reply:
I have updated to use #{$bedrock-module-path} instead. I believe you are correct that we will need to make some adjustments to use $bedrock-module-path in `bedrock-views`.